configs/run-wrapper_config.yaml

```diff
@@ -9,16 +9,18 @@ novaseq_A01736:
   demultiplex_path: "/seqstore/novaseq_A01736/Demultiplexdir/"
   seq_name_regex: '^[0-9]{6}_A01736_[0-9]{4}_.{10}$'
 
-previous_runs_file_path: 'novaseq_runlist.txt'
+previous_runs_file_path: 'novaseq_runlist_test.txt'
 
 hg38ref:
   GMS-BT: 'yes'
   GMS-AL: 'yes'
 
 workingdir: '/medstore/results/wgs/wgs_somatic'
 
+hcp_download_dir: '/medstore/projects/P23-075/wgs_somatic/hcp_downloads'
```

Comment:
cant we think of a better location?

Reply:
dont fix right now

Reply:
I would need to ask IT to create the folder for '/medstore/tmp/hcp_downloads', this was just to get the code looked at but if everything looks alright then I can ask IT about it

Reply:
We can just decide. /medstore/tmp is not good however